domains/juan-samayoa.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "juansamayoa"
+    },
+    "record": {
+        "RECORDTYPE": "URL"
```

Comment:
Invalid RECORDTYPE and the URL should be actual values. See docs: https://docs.is-a.dev